packages/contracts/contracts/optimistic-ethereum/OVM/bridge/messaging/OVM_L1CrossDomainMessenger.sol

```diff
@@ -51,11 +48,24 @@ contract OVM_L1CrossDomainMessenger is
         bytes32 indexed _xDomainCalldataHash
     );
 
+    /*************
+     * Constants *
+     *************/
+
+    // The default x-domain message sender being set to a non-zero value makes
+    // deployment a bit more expensive, but in exchange the refund on every call to
+    // `relayMessage` by the L1 and L2 messengers will be higher.
+    address internal constant DEFAULT_XDOMAIN_SENDER = 0x000000000000000000000000000000000000dEaD;
```

Comment:
Nit: in the [EM we use](https://github.com/ethereum-optimism/optimism/blob/8168765ec4625d96978214c40daa0b88f5fb25c7/packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol#L73): `0xdEfa017defA017DeFA017DEfa017DeFA017DeFa0`.

Here I think we should use the same value (OR use this value in the EM). It would also be nice to move this (as well as the `DEFAULT_UINT256` from the EM) into a small library, but that might be better in a separate PR.

Reply:
Logged https://github.com/ethereum-optimism/optimism/issues/803 to track this. I agree we should be consistent.